packages/os/ephemeral-storage.rules

```diff
@@ -0,0 +1,27 @@
+# ephemeral storage links: /dev/disk/ephemeral
+
+ACTION=="remove", GOTO="ephemeral_storage_end"
```

Comment:
Do we want to continue for `change` actions?  `ACTION!="add"` may be more accurate?

Reply:
We need `change` since that fires when the partition table is rewritten, which can influence whether a device is counted as an ephemeral.

There are a lot of other actions that we don't care about. `60-persistent-storage.rules` also uses this `ACTION=="remove"` guard, so I think it's still the right choice.